src/plugins/data_source/server/data_source_service.ts

```diff
@@ -64,6 +64,8 @@ export class DataSourceService {
       };
     };
 
+    // TODO: consider to expose pool operation. For example, remove a client from pool when delete datasource
```

Comment:
Do we have an issue created for the ToDo?

Reply:
I'll remove the todo, it doesn't provide much context with comment in the code. Will create issue